lang/cs/Org.Apache.REEF.Network/Group/Driver/Impl/GroupCommDriver.cs

```diff
@@ -147,6 +148,24 @@ public ICommunicationGroupDriver NewCommunicationGroup(string groupName, int num
         }
 
         /// <summary>
+        /// Remove a group from the GroupCommDriver
+        /// Throw ArgumentException if the group does not exist
+        /// </summary>
+        /// <param name="groupName"></param>
+        public void RemoveCommunicationGroup(string groupName)
+        {
+            lock (_groupsLock)
+            {
+                if (!_commGroups.ContainsKey(groupName))
+                {
+                    Exceptions.Throw(new ArgumentException("Group Name is not registered with GroupCommunicationDriver"), LOGGER);
```

Comment:
@jwang98052 I'm actually not sure about that. Do we have a `CaughtAndThrow` outside of this block? We wouldn't want excessive logging of the same `Exception` polluting our logs.